CHANGELOG_PENDING.md

```diff
@@ -1,19 +1,35 @@
 # Pending
 
+## v0.26.0
+
+*October 19, 2018*
+
 Special thanks to external contributors on this release:
-@goolAdapter, @bradyjoestar
+@goolAdapter, @bradyjoestar, @HaoyangLiu, @Slamper, @james-ray,
+@overbool, @connorwstein, @phymbert, @Uzair1995
+
+This release is primarily about adding Version fields to various data structures,
+optimizing consensus messages for signing and verification in
+restricted environments (like HSMs and the Ethereum Virtual Machine), and
+aligning the consensus code with the [specification](https://arxiv.org/abs/1807.04938).
+It also includes our first take at a generalized merkle proof system.
```

Comment:
May be worth mentioning limiting total number of evidences per block? (Thats in the changelog, though I don't remember a PR switching to number instead of proportion)

Reply:
We haven't made that switch yet. Not sure it's a high level concern worth pointing out here.